config-model-api/src/main/java/com/yahoo/config/application/api/DeploymentSpec.java

```diff
@@ -46,7 +46,8 @@
                                                                   "<deployment version='1.0'/>",
                                                                   Optional.empty(),
                                                                   Optional.empty(),
-                                                                  Notifications.none());
+                                                                  Notifications.none(),
+                                                                  Collections.emptyList());
```

Comment:
Tip: Use `List.of()`, `Set.of()` etc. for empty collections. Shorter to read and typically allows skipping the `java.util.Collections` import altogether.